src/lib/select/select.spec.ts

```diff
@@ -449,6 +450,22 @@ describe('MdSelect', () => {
       expect(fixture.componentInstance.select.selected).not.toBeDefined();
     });
 
+    it('should be able to programmatically select a falsy option', () => {
```

Comment:
Nit: Why is this in its own section? Seems like it would better fit under forms integration since it has to do with setting the form control value.